test/dynamo/test_allow_inline_skip.py

```diff
@@ -0,0 +1,47 @@
+# Owner(s): ["module: dynamo"]
+import unittest
+
+import torch
+import torch._dynamo.test_case
+from torch._dynamo.utils import istype
+
+
+class AllowInlineSkipTests(torch._dynamo.test_case.TestCase):
+    def test_func_inlinelist(self):
+        def fn(x):
+            if istype(x, torch.Tensor):
+                return x + 1
+            else:
+                return x - 1
+
+        my_func_inlinelist = torch._dynamo.skipfiles.FUNC_INLINELIST.copy()
+        my_func_inlinelist.add("torch._dynamo.utils.istype")
+
+        def my_get_func_inlinelist():
+            inlinelist = set()
+            for f in my_func_inlinelist:
+                inlinelist.add(eval(f).__code__)
+            return inlinelist
+
+        self.assertTrue(
+            "torch._dynamo.utils" not in torch._dynamo.skipfiles.FILE_INLINELIST
```

Comment:
not sure what you're trying to test here

Reply:
Look at this test, I'd like to check the function (```torch._dynamo.utils.istype```) being inlined correctly if adding it to ```FUNC_INLINELIST```. However, we should first check the function's file (```torch._dynamo.utils```) is not in the ```FILE_INLINELIST```, otherwise, it will be inlined even the ```FUNC_INLINELIST``` doesn't work well.

Reply:
I'd propose you make a "new module" in the fly that is not in torch namespace and test with that. Stuff in torch is too brittle.

Reply:
Yea, I can add a test like this.